ws-nextjs-app/cypress/e2e/languagesPage/assertions/index.ts

```diff
@@ -1,11 +1,24 @@
-export const assertPidginHomepage = () => {
-  it('should render the Pidgin Homepage', () => {
-    cy.get('h1').should('have.text', 'BBC News, Pidgin - Home');
-    cy.get('h1').should('not.have.text', 'Get the news in your language');
-  });
-};
 export const assertWSLanguagesPage = () => {
-  it('should render the Languages Page', () => {
+  it('should render the WS Languages Page', () => {
     cy.get('h1').should('have.text', 'Get the news in your language');
   });
 };
+
+export const assertWSLanguagesPageLocal = () => {
+  it('should render the Languages Page with fixture content', () => {
+    cy.contains('BBC World Service - global front page').should('exist');
+  });
+};
+
+export const assertWSLanguagesPageURN = () => {
+  it('should render the WS Languages Page with correct URN', () => {
+    cy.window().then(win => {
+      const windowData = (
+        win as Window & { SIMORGH_DATA?: { pageData?: { urn?: string } } }
+      ).SIMORGH_DATA;
+      const pageData = windowData?.pageData;
```

Comment:
We have a helper utility for this: `cy.getPageDataFromWindow` - https://github.com/bbc/simorgh/blob/11f33319a2d168719d88a9b57e519e9525c69e58/cypress/e2e/pages/liveRadio/tests.js#L55-L56